README.md

```diff
@@ -33,6 +33,11 @@ firebase = Firebase::Client.new(base_uri, secret_key)
 response = firebase.push("todos", { :name => 'Pick the milk', :priority => 1 })
 ```
 
+If you have to configure HTTP Client timeout, set times as follows:
```

Comment:
This kind of reads like ESL, I would explain what each of those options do, or link to the appropriate page in httpclient's documentation.